src/components/DarkMode/DarkMode.component.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: inline-block;
+  padding: 0 20px;
+  height: 40px;
+`;
+const Switch = styled.div`
+  display: inline-block;
+  position: relative;
+  margin: 0 20px;
+`;
+const Track = styled.div`
+  display: block;
+  width: 40px;
+  height: 10px;
+  background-color: #ffffff;
+  border-radius: 100px;
+`;
+const Ball = styled.div`
```

Comment:
This looks good, you can improve it by moving the styled components to their own files, something like `DarkMode.styled.jsx`, and keep in this file only the logic and the component markup